tests/js/server/recovery/view-search-alias-populate-no-sync.js

```diff
@@ -48,11 +48,31 @@ function runSetup () {
   internal.wal.flush(true, true);
   internal.debugSetFailAt("FlushCrashBeforeSyncingMinTick");
 
-  for (let i = 0; i < 50000; i++) {
-    c.save({ a: "foo_" + i, b: "bar_" + i, c: i });
+  if (global.hasOwnProperty('arango')) {
+    // we intend to crash, so we should get to know quickly:
+    print('setting short timeout');
+    arango.timeout(1);
+  }
```

Comment:
hmm, but what if crash not really happened and we just faced with slow requests in our CI?

Maybe remove it?
It's just optimization

Reply:
I'd say 1s still is plenty, right?